.github/workflows/push-to-npm-gpr.yml

```diff
@@ -28,3 +28,15 @@ jobs:
         run: npm publish --access public mitre-saf-v*.tgz
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      # Setup .npmrc file to publish to GitHub Package Registry
+      - uses: actions/setup-node@v3
+        with:
+          registry-url: 'https://npm.pkg.github.com'
+          scope: '@mitre'
+
+      # Publish SAF CLI to GitHub Package Registry
+      - name: Publish SAF CLI to GPR
+        run: npm publish mitre-saf-v*.tgz
```

Comment:
"When publishing scoped packages, the access level defaults to restricted. If you want your scoped package to be publicly viewable (and installable) set --access=public." - https://docs.npmjs.com/cli/v8/commands/npm-publish#access

idk why we need `--access public` for the npm one since it doesn't have that scope input, but shouldn't this publish call have it then?